pkg/config/multizone/kds.go

```diff
@@ -15,6 +15,8 @@ type KdsServerConfig struct {
 	RefreshInterval config_types.Duration `json:"refreshInterval" envconfig:"kuma_multizone_global_kds_refresh_interval"`
 	// Interval for flushing Zone Insights (stats of multi-zone communication)
 	ZoneInsightFlushInterval config_types.Duration `json:"zoneInsightFlushInterval" envconfig:"kuma_multizone_global_kds_zone_insight_flush_interval"`
+	// TlsDisable turns off TLS for KDS
+	TlsDisable bool `json:"tlsDisable" envconfig:"kuma_multizone_global_kds_tls_disable"`
```

Comment:
it's either this or we need to make the field optional https://github.com/kumahq/kuma/blob/18150a5619b6ef61fc048169160d9d8e1d0c37b3/pkg/core/bootstrap/autoconfig.go#L66-L69 because it's always configured via default